test/cbmc/proofs/requestJob_Mqtt/requestJob_Mqtt_harness.c

```diff
@@ -95,12 +98,16 @@ void requestJob_Mqtt_harness()
 {
     OtaAgentContext_t agent;
     OtaInterfaces_t otaInterface;
+    size_t size;
+
+    __CPROVER_assume( size >= 1 && size < 128U );
```

Comment:
Could you add a comment on why is safe to assume this range for this size?

Reply:
I'll complete this in a follow up PR which will expand the readme to include testing steps.